lua/gcompute/ui/ide/plugins/metastruct_coh.lua

```diff
@@ -98,12 +98,11 @@ function self:UpdateChatStatus ()
 		if #code > 4096 then
 			code = string.sub (code, 1, GLib.UTF8.GetSequenceStart (code, 4097) - 1) .. "..."
 		end
-		local _, n = code:gsub("\n", '\n')
-		local title = code:match("^[\n%s]*([^\n]+)") or ""
-		code = ( 
-			not string.find(code,"secret",1,true) and 
-			not string.find(code,"SECRET",1,true)
-		) and code or ("[Coding Lua (%d lines)]\n%s"):format(n, title)
+		local _, n = string.gsub (code, "\n", "\n")
+		local title = string.match (code, "^[\n%s]*([^\n]+)") or ""
```

Comment:
Move this inside the if statement
`%s` covers `\n`
Not all line breaks are `\n`